core/src/main/resources/hudson/cli/Messages_de.properties

```diff
@@ -1,11 +1,65 @@
 CancelQuietDownCommand.ShortDescription=Wirkung des Befehls "quiet-down" wieder aufheben.
-OfflineNodeCommand.ShortDescription=Knoten wird bis zum n\u00e4chsten "online-node"-Kommando f\u00fcr keine neuen Builds verwendet.
+OfflineNodeCommand.ShortDescription=Knoten wird bis zum n\u00E4chsten "online-node"-Kommando f\u00FCr keine neuen Builds verwendet.
 
+LoginCommand.ShortDescription=Speichert die aktuellen Anmeldedaten, damit weitere Befehle ohne gesonderte Anmeldung ausgef\u00FChrt werden k\u00F6nnen.
+BuildCommand.CLICause.CannotBuildUnknownReasons=Kann {0} aus unbekannten Gr\u00FCnden nicht bauen.
+HelpCommand.ShortDescription=Zeigt eine Auflistung aller verf\u00FCgbaren Befehle oder die detaillierte Beschreibung eines einzelnen Befehls.
+InstallPluginCommand.DidYouMean={0} sieht wie ein Plugin-Kurzname aus. Meinten Sie \u2018{1}\u2019?
+UpdateNodeCommand.ShortDescription=Aktualisiert die Konfiguration eines Knoten basierend auf der via Standardeingabe (stdin) bereitgestellten XML-Repr\u00E4sentation. Das Gegenst\u00FCck zum Befehl get-node.
+InstallPluginCommand.InstallingPluginFromUrl=Installiere ein Plugin von {0}
+GroovyCommand.ShortDescription=F\u00FChrt das angegebene Groovy-Script aus.
+SetBuildResultCommand.ShortDescription=Setzt das Ergebnis des aktuellen Builds. Dieser Befehl funktioniert nur innerhalb eines Builds.
+DeleteBuildsCommand.ShortDescription=L\u00F6scht Builds.
+ListChangesCommand.ShortDescription=Zeigt das Changelog des angegebenen Builds.
+BuildCommand.ShortDescription=Startet einen Build, und wartet optional auf dessen Ende.
+ReloadJobCommand.ShortDescription=L\u00E4dt einen Job neu.
+WhoAmICommand.ShortDescription=Zeigt Ihren Login und Berechtigungen an.
+BuildCommand.CLICause.CannotBuildDisabled={0} kann nicht gebaut werden, da es deaktiviert ist.
+UpdateViewCommand.ShortDescription=Aktualisiert die Konfiguration einer Ansicht basierend auf der via Standardeingabe (stdin) bereitgestellten XML-Repr\u00E4sentation. Das Gegenst\u00FCck zum Befehl get-view.
+CreateJobCommand.ShortDescription=Erstellt einen neuen Job basierend auf der via Standardeingabe (stdin) bereitgestellten XML-Repr\u00E4sentation.
+InstallToolCommand.ShortDescription=Installiert ein Hilfsprogramm und gibt das Installationsverzeichnis aus. Dieser Befehl funktioniert nur innerhalb eines Builds.
+GetViewCommand.ShortDescription=Gibt die XML-Repr\u00E4sentation der angegebenen Ansicht aus.
+SetBuildDescriptionCommand.ShortDescription=Setzt die Beschreibung eines Builds.
+AddJobToViewCommand.ShortDescription=F\u00FCgt Jobs zu einer Ansicht hinzu.
+ListJobsCommand.ShortDescription=Zeigt alle Jobs, die zu einer Ansicht oder einem Ordner geh\u00F6ren.
+GetJobCommand.ShortDescription=Gibt die XML-Repr\u00E4sentation des angegebenen Jobs aus.
+CliProtocol2.displayName=Jenkins CLI Protocol/2 (Verschl\u00FCsselung der Transportschicht)
```

Comment:
I mentioned it as "Protokoll" was used in a bunch of other places in reference to the CLI/JNLP protocol.